Helper/Data.php

```diff
@@ -1853,4 +1853,27 @@ public function getCurrentLocaleCode($storeId)
 
         return $localeCode;
     }
+
+    /**
+     * Get the Customer Area PSP Search URL with a preset PSP Reference
+     *
+     * @param string $pspReference
+     * @param string $liveEnvironment
+     * @return string
+     */
+    public function getPspReferenceSearchUrl($pspReference, $liveEnvironment)
+    {
+
+        if ($liveEnvironment === "true") {
```

Comment:
There is a function to get the checkout environment in the data helper: getCheckoutEnvironment - you can use this instead of the if else